### DISCUSSION TITLE
邀请社区积极分子给新进菜鸟做普及教育

### DISCUSSION THREAD

**Discussion by lxm218** [tearust/Chinese-wiki#10] [Ideas]
鉴于TEA Project的learning curve比较大，新人问问多多，邀请社区积极分子给新进菜鸟做普及教育，欢迎大家讨论以什么方式来推荐这个事情。

**Comment by kevingzhang** [comment]
我设想的方法其实就是已经刚刚开始有一个雏形的TEA Party.
TEA Party再次出现在epoch 7 的时候就是一个可以带有支付接口的社交TApp了.
这支付接口可以实现有偿提问回答.
我们会建立一个NPC自己创建的Help Desk TEA Party 应用.
提问者支付押金. 回答者得到提问者认可正确答案就可以拿走这笔押金.
如果这个问题被NPC认定为有价值的提问和回答, 就可以PIN. PIN的话由NPC支付存储费用并且给提问和回答者双重奖励.

而且每一个新用户会得到一笔学费奖励. 他们不花也销毁了. 这笔钱只能用于在Help Desk问问题. 解答者拿走这些奖金.

**Comment by kevingzhang** [comment]
因为Help Desk是一个NPC建立的应用. 所以NPC不需要挣钱. 作为一个公益应用. Theta可以设置为0. 然后支付固定TEA给Hosting的节点.

这样便于冷启动.

**Comment by kevingzhang** [comment]
矿工乐意host这样一个应用. 因为旱涝保收. 这样至少可以持续运行下去. 然后提问者花NPC给的补助学费来提问. 老鸟们回答问题获得TEA奖励. 这些TEA随后在每周的兑换CML中就有机会换的真的CML NFT.
如果提问得当, NPC认可成为系统置顶话题(就是不会删除的那种), 提问者和回答者都会获得额外的NPC奖励的TEA.

这样下来就通过NPC补助形成大家提问回答的社区氛围.
中文和英文可以分开运作.

**Comment by kevingzhang** [comment]
具体产品设计大家请献计献策. 可以学习现有的web2 的问答社区软件. 不过需要考虑资金奖励这种web3才有的特色内容. 要把经济激励加进去. Web2是一个假借乌托邦理想实则偷换概念把用户隐私偷走换钱的商业模式. Web3则是一个实打实的资本主义模式. 人人在满足私利的前提下帮助其他人, 实现共同富裕.

**Comment by pzjboy0924** [comment]
对于积极参与社区建设、wiki贡献等等，请你们也考虑一下打赏机制！

**Reply by kevingzhang** [reply]
已经在产品设计中了. 看上面我的答复就提到了

**Comment by lxm218** [comment]
Helpdesk TEA Party 初步设计：
1. NPC 公益TApp，theta为0，Host fee为fixed TEA，没有收益分红；
2. 每个user获得1500T credit专用于HelpDesk TEA Party，不能withdraw and transfer；credit存在一个专用credit account；其他T（包括收益获得的）存在另一个可以withdraw and transfer的TEA account；
3. 每次提问押金50T；
4. 允许多人回答一个问题；
4. 答问者获得T的数量由提问者决定；
5. 答问者在回答问题之前可以看见该问题的剩余押金,以此决定是否回答问题该问题；
6. NPC选择有价值的问题和答案进行Pin，被pin的问搭将获得100T的奖励，由提问者和问题者分享，进入各自的TEA account；
